test/e2e/specs/editor/various/manage-reusable-blocks.test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Managing reusable blocks', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.visitAdminPage( 'edit.php', 'post_type=wp_block' );
+	} );
+
+	test( 'Should import reusable blocks', async ( { admin, page } ) => {
+		async function getNumberOfEntries() {
+			return page.evaluate(
+				() => document.querySelectorAll( '.hentry' ).length
+			);
+		}
+
+		const originalEntries = await getNumberOfEntries();
```

Comment:
Bummer that there isn't a good enough accessible selector for the entry. 😞 

```suggestion
		const originalEntries = await page.locator( '.hentry' ).count();
```